docs/best-practices/crate-node.txt

```diff
@@ -0,0 +1,460 @@
+so if the cluster is still available the node can be simply replaced by
+a fresh node on a new host which will replace the broken node.
+
+crate-node
+==========
+
+.. code-block:: console
+
+   bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
+   [--ordinal <Integer>] [-C <KeyValuePair>]
+   [-h, --help] ([-s, --silent] | [-v, --verbose])
+
+
+The following commands provide a various operations to modify and recover nodes:
+
+* ``crate-node repurpose`` can be used to delete unwanted data from a node if it
+  used to be a data node or a master-eligible node but has been repurposed not to
+  have one or other of these roles.
+
+* ``crate-node unsafe-bootstrap`` can be used to perform unsafe cluster bootstrapping.
+  It forces one of the nodes to form a brand-new cluster on its own, using its local
+  copy of the cluster metadata.
+
+* ``crate-node detach-cluster`` enables you to move nodes from one cluster to another.
+  This can be used to move nodes into a new cluster created with the
+  ``crate-node unsafe-bootstap`` command. If unsafe cluster bootstrapping was not possible,
+  it also enables you to move nodes into a brand-new cluster.
+
+Changing the role of a node
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Certain situations may require to change the initial type of a node. The
+``crate-node repurpose`` command can modify and delete the data stored on a  disk
+from a node.
+
+The intended use is:
+
+* Stop the node
+
+* Update crate.yml by setting node.master and node.data as desired.
+
+* Run ``crate-node repurpose`` on the node
+
+* Start the node
+
+if you run ``crate-node repurpose`` on a node with ``node.data: false`` and
+``node.master: true`` then it will delete any remaining shard data on that node,
+but it will leave the table and cluster metadata alone.
+
+If you run ``crate-node repurpose`` on a node with ``node.data: false`` and
+``node.master: false`` then it will delete any remaining shard data and table
+metadata, but it will leave the cluster metadata alone.
+
+**Change the type from a data to become a master node**
+
+When a data-node should be repurposed as master-node the following steps are needed:
+- Change the nodes settings in crate.yml ``node.master: true`` and ``node.data: false``
+
+
+Change the type to a coordination node
+
+When a data-node should be repurposed as master-node the following steps are needed:
+- Change the nodes settings in crate.yml ``node.master: false`` and ``node.data: false``
+
+.. WARNING::
+
+  Running this command can lead to data loss if the locally stored data from the
+  node is not persisted on the nodes in rest of the cluster.
+
+The tool provides a summary of the data to be deleted and asks for confirmation
+before making any changes. You can get detailed information about the affected
+tables and shards by passing the verbose (-v) option.
+
+Recovering data after a disaster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes CrateDB nodes are temporarily stopped, perhaps because of the need
```

Comment:
Similar here and in the following paragraphs, I think this could be more compressed and more to the point.

The amount of weasel words could also be reduced. `very rare` -> `rare`.  `serious disaster` -> `disaster`, etc..